plugins/systems/move/Move.cpp

```diff
@@ -43,7 +43,7 @@ void Systems::MoveSystem::run(Engine::GameEngine &engine)
                 vel->floatY *= factor;
                 vel->x = (int)vel->floatX;
                 vel->y = (int)vel->floatY;
-                engine.updateComponent(i, vel->getId(), vel->serialize());
+                // engine.updateComponent(i, vel->getId(), vel->serialize());
```

Comment:
Is this comment intentionnal ?

Reply:
Yes it is, we dont need to send data about the velocity component to the client. He doesn't care about it ^^ !

The only components he cares about are : 
- Action
- SpriteId
- Position

Reply:
Okay :saluting_face: